_includes/head.html

```diff
@@ -8,7 +8,7 @@
 	<meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
 
 	<meta property="og:type" content="website">
-	<meta property="og:url" content="http://code4sa.org{{ page.url }}">
+	<meta property="og:url" content="{{ page.url }}">
```

Comment:
Keep this. I don't think page.url is a complete URL and, like the image URL, it needs to be.